.github/workflows/build.yml

```diff
@@ -5,6 +5,12 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-20.04
+    env:
+      TEST_MAPSCRIPT_PYTHON: 'true'
```

Comment:
you need to explicitly pass them with the -e option of docker run

Reply:
Also outdated.